assets/dev/scss/editor/panel/controls/_gap.scss

```diff
@@ -0,0 +1,104 @@
+.elementor-control-type-gap {
```

Comment:
This entire CSS file looks like a copy of `_dimensions.scss`. Maybe we can extract the common code to `_proportions_common.scss` and then make `_gap.scss` and `_dimensions.scss` individually much smaller and have them import the common file. What do you think?

Reply:
Hi, What I ended up doing was to inherit the `dimensions.scss` into the `gap.scss`.
I thought of creating a common base, the issue is that this is scss, and the main wrapper class is different, therefor AFAIU, the base should have the control's entire code. and then in each of the individual files I need to override the control-related parts.
Please correct me if I am wrong